scripts/integration/datadog-e2e-logs/compose.yaml

```diff
@@ -0,0 +1,98 @@
+version: '3'
```

Comment:
I definitely agree. In fact, In the RFC I outlined making a separate subcommand in vdev to invoke them (like `vdev e2e` (as opposed to `vdev integration`).  Regarding the vdev changes, because I'm re-using basically the entire integration test infrastructure there, we could potentially do a small re-factor to allow the two subcommands to exist and not have duplicate code, but I felt there were enough code changes here that it deserved a separate PR .

Perhaps a small compromise would just be to move these to `scripts/e2e` as you say, but then in the vdev int test code path, just add that directory as one to look for test cases within. And create a followup GH issue to track the refactor to add `vdev e2e` subcommand. 

WDYT?

Reply:
Created https://github.com/vectordotdev/vector/issues/18829